design/media_types.go

```diff
@@ -62,9 +62,10 @@ var meta = a.Type("workItemListResponseMeta", func() {
 // position represents the ID of the workitem above which the to-be-reordered workitem(s) should be placed
 var position = a.Type("workItemReorderPosition", func() {
 	a.Description("Position represents the ID of the workitem above which the to-be-reordered workitem(s) should be placed")
-	a.Attribute("above", d.String, "ID of the workitem above which the to-be-reordered workitem(s) should be placed")
+	a.Attribute("id", d.String, "ID of the workitem above which the to-be-reordered workitem(s) should be placed")
+	a.Attribute("direction", d.String, "position of workitem")
```

Comment:
If this really is a direction parameter, it might make sense to define it like this:

```go
a.Attribute("direction", d.String, "...", func() {
    a.Enum("above", "below")
})
```

Alternatively if the string MUST NOT be empty, you can define it like so:

```go
a.Attribute("direction", d.String, "...", func() {
    a.MinLength(1)
})
```

Then you don't need to check for empty strings in the controller. The same is true (at least for `a.MinLength(1)`) for the `id` attribute.